zerver/management/commands/soft_deactivate_users.py

```diff
@@ -76,11 +74,11 @@ def handle(self, *args: Any, **options: str) -> None:
                 users_to_deactivate = list(users_to_deactivate)
 
                 if len(users_to_deactivate) != len(user_emails):
-                    user_emails_found = [user.email for user in users_to_deactivate]
-                    for user in user_emails:
-                        if user not in user_emails_found:
-                            raise Exception('User with email %s was not found. '
-                                            'Check if the email is correct.' % (user,))
+                    user_emails_found = {user.email for user in users_to_deactivate}
+                    user_emails_not_found = ', '.join(set(user_emails) - user_emails_found)
+                    raise Exception('Users with emails %s were not found. '
+                                    'Check if they are correct.' % (user_emails_not_found))
```

Comment:
We might want to print the emails one-per line if we thought this would be a common code path with multiple emails, but I think it really isn't given that the default flow is auto-detection.

Reply:
Raising an exception once per invalid email ID would be annoying if there were more than one invalid email IDs supplied - the user would've to run the command multiple times.  

I could change this to print the email IDs one per line.

Reply:
yeah, one per line would be "better" (and we should raise `CommandError` to proper management command styling).  But as I mentioned, I think realistically the use case of more-than-one directly specified email is likely to be very uncommon, so I'm also fine with punting on doing a nicer display.